pkg/appmanager/as3Manager.go

```diff
@@ -104,12 +106,34 @@ func (appMgr *Manager) processUserDefinedAS3(template string) bool {
 // Validates the AS3 Template
 func (appMgr *Manager) validateAS3Template(template string) bool {
 
-	var schema = appMgr.schemaLocal + "as3-schema-3.11.0-3-cis.json"
+	res, _ := http.Get(as3SchemaLatest)
```

Comment:
Let's catch and handle the error.

Reply:
Handling the error if occurred while fetching the latest.

Named err parameter as resErr to avoid shadowing of variable.